### DISCUSSION TITLE
Skipping download of remote folder in OneDrive

### DISCUSSION THREAD

**Discussion by nmarks99** [abraunegg/onedrive#1784] [Q&A] [answered]
Hi, I am trying to ignore sync for certain folders in my OneDrive remote folder. For example, in my OneDrive, I have the folders A, B, C, and D but when I run the onedrive command, I only want to sync folders A and B and leave B and C only in the cloud and completely ignore them. Is this possible? So far I have tried putting a .nosync file in C and D folders that are in OneDrive before syncing but that still resulted in folders C and D getting downloaded locally. Just to reiterate, I want folders C and D to remain in the cloud in my OneDrive account, but to never be downloaded locally on my computer.

Thanks in advance for the help

**Comment by abraunegg** [comment, marked as answer]
@nmarks99 
The `.nosync` option to skip files is only meant to be used with local folders - placing that file in remote folders will not achieve the desired outcome you are looking for.

You need to investigate 'sync_list' to achieve what you are looking to do.

**Reply by nmarks99** [reply]
It seems that sync_list also only works with local folders. As a test, I added folder "A" to the sync_list, and expected it then to only download folder A from OneDrive cloud, however it did not download anything. I assume this is because it was looking for a folder in ~/OneDrive named "A". To reiterate, what I would like to do is download only specific folders from the cloud, assuming I am starting with an empty ~/OneDrive directory.

**Comment by abraunegg** [comment]
@nmarks99 
'sync_list' is for local and remote folders - it most certainly gets applied and used when downloading content from OneDrive.

Suggest that you do the following:
1. Ensure you are running the latest version of the client
2. Do not enable any systemd service. If you are installing from a package on Ubuntu or any system derived from it, make sure you read and understand the known issues of using those packages: https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md#known-issues-with-installing-from-the-above-packages
3. Run the client manually using `--dry-run --download-only --verbose` so you can see what is going on, what will happen based on your configuration.
4. Once you are comfortable with that will happen, drop `--dry-run` and download your data

**Reply by nmarks99** [reply]
I'll give this a try thanks for the help!
